### DISCUSSION TITLE
HA non raggiungibile

### DISCUSSION THREAD

**Discussion by FlavioRadice** [indomus/forum#2800] [HUB: Home Assistant]
Cuiao, capita spesso che HA non sia più raggiungibile dalla rete interna, sia dal pc (windows) che fa girare l'istanza , che dalla rete esterna.

Non riuscendo a prendere il ocntrollo non posso fare altro che riavviare la VM su Oracle Virtualbox, ma non riesco a capire se è HA o Virtualbox che doventano non piu raggungibili.

Su internet avevo anche letto di disabilitare il protocollo IPv6 e l'ho fatto su HA

Chi può aiutarmi? Grazie

**Comment by popeye70** [comment]
Ciao, anche io ho un problema simile: saltuariamente HA diventa inaccessibile sia da browser che da app, nonostante nella Virtualization Station del mio Qnap la sua VM risulta in esecuzione e il router lo rilevi presente in rete.
L'unica apparente soluzione è quindi forzare lo spegnimento della VM e lanciarne il riavvio, che però spesso si blocca con errori di timeout del Supervisor oppure quando HA riesce a ripartire trovo non caricate la maggior parte delle integrazioni previste e nel registro leggo una serie di avvisi ed errori tipo questi
[home-assistant_2024-01-14T09-15-10.112Z.log](https://github.com/indomus/forum/files/13930900/home-assistant_2024-01-14T09-15-10.112Z.log)

Qualcuno gentilmente riesce ad aiutarmi a capire cosa sta succedendo?
Grazie
